defaults/main.yml

```diff
@@ -12,16 +12,27 @@ humio_data_secondary_path: ~
 humio_permissions: ~
 humio_processes: "{{ ansible_local.numanodes | length }}"
 
-humio_java_opts_Xms: 4G
-humio_java_opts_Xmx: 32G
+# Default memory options configured for a machine with 64G of
+# memory. You can adjust the default by overriding `humio_total_memory`
+# or for more fine-grained control you can override the
+# `humio_java_opts_*` variables directly. See the docs for more details
+# about JVM memory usage:
+# https://docs.humio.com/operations-guide/configuration/basic-configuration/jvm-configuration/#java-memory-options
+humio_total_memory: 64
```

Comment:
I prefer adding a suffix to variable names that indicates the unit. In this case I would probably go with something like `humio_total_memory_gb`.

Did you consider using Ansible facts to get default values for memory (and potentially also CPU)? Seems to have a few things available to use:
```
    "ansible_memtotal_mb": 128632,
    "ansible_processor_cores": 8,
    "ansible_processor_count": 1,
    "ansible_processor_threads_per_core": 2,
    "ansible_processor_vcpus": 16,
```

Reply:
I hadn't actually, but that's not a bad idea. I don't know that the built-in processor counts will work how we need since we rely on numa node counts, which I don't think is included in default facts. But we already have a script extracting that from the system, so processor count should be correct based on the system it's running on already.

Reply:
Swapped it to using megabytes vs gigabytes so the math works out cleaner. @SaaldjorMike could you take another peek?